client/src/elm/Translate.elm

```diff
@@ -11518,7 +11518,7 @@ translationSet trans =
 
                 TreatmentErythromycin ->
                     { english = "by mouth 4x a day for 14 days"
-                    , kinyarwanda = Just "mu kanwa inshuro enye ku munsi mu minsi 14"
+                    , kinyarwanda = Just "Mu kanwa inshuro enye ku munsi mu minsi 14"
```

Comment:
I think that whatever starts with small letter in English should start with small letter as well.
If I'm right, change, if not, ignore.